webapi/src/test/scala/org/knora/webapi/e2e/v1/ResourcesV1R2RSpec.scala

```diff
@@ -1471,6 +1471,29 @@ class ResourcesV1R2RSpec extends R2RSpec {
             }
         }
 
+        "consider inherited cardinalities when generating XML schemas for referenced ontologies in an XML import" in {
+            val ontologyIri = URLEncoder.encode("http://www.knora.org/ontology/something", "UTF-8")
+
+            Get(s"/v1/resources/xmlimportschemas/$ontologyIri") ~> addCredentials(BasicHttpCredentials(biblioUserEmail, password)) ~> resourcesPath ~> check {
+                val responseBodyFuture: Future[Array[Byte]] = response.entity.toStrict(5.seconds).map(_.data.toArray)
+                val responseBytes: Array[Byte] = Await.result(responseBodyFuture, 5.seconds)
+                val zippedFilenames = collection.mutable.Set.empty[String]
+
+                for (zipInputStream <- managed(new ZipInputStream(new ByteArrayInputStream(responseBytes)))) {
+                    var zipEntry: ZipEntry = null
+
+                    while ( {
+                        zipEntry = zipInputStream.getNextEntry
+                        zipEntry != null
+                    }) {
+                        zippedFilenames.add(zipEntry.getName)
+                    }
+                }
+
+                assert(zippedFilenames == Set("something.xsd", "knoraXmlImport.xsd", "anything.xsd"))
```

Comment:
Don't you have to look at the files' contents?

Reply:
That would be nice but I was too lazy. :)

Reply:
Is there anything you cannot predict (like internal, temporary ids)? Otherwise couldn't you just use the xml diff we already have?

Reply:
I think the schemas should be totally predictable, I'll add something to check them. Feeling less lazy now that I've had a cup of coffee. :)